pokemongo_bot/__init__.py

```diff
@@ -204,11 +236,15 @@ def work_on_cell(self, cell, position):
                            0], self.position[1], x['latitude'], x['longitude']))
 
                 for fort in forts:
-                    worker = MoveToFortWorker(fort, self)
-                    worker.work()
-
-                    worker = SeenFortWorker(fort, self)
-                    hack_chain = worker.work()
+                    worker = MoveToFortWorker(fort, self, cell)
+                    hack_chain = None;
+                    while True:
```

Comment:
There shouldn't be any loops here. `_work_on_cell` is called repeatedly and *is* the loop. This should only take a single step towards the nearest fort, or spin a single fort if it is near enough. Then we will go through _work_on_cell again and it will run for the next fort. That will allow us to catch pokemon between every fort.